extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -492,25 +497,61 @@ export class AmpLightboxGallery extends AMP.BaseElement {
    * @param {!Event} e
    * @private
    */
-  toggleControls_(e) {
+  onToggleControls_(e) {
     if (!this.shouldTriggerClick_(e)) {
       return;
     }
-
     if (this.controlsMode_ == LightboxControlsModes.CONTROLS_HIDDEN) {
-      this.carousel_.setAttribute('controls', '');
-      this.topBar_.classList.remove('fade-out');
-      if (!this.container_.hasAttribute('gallery-view')) {
-        this.descriptionBox_.classList.remove('fade-out');
-        this.updateDescriptionBox_();
-      }
-      this.controlsMode_ = LightboxControlsModes.CONTROLS_DISPLAYED;
+      this.showControls_();
     } else {
-      this.carousel_.removeAttribute('controls');
+      this.hideControls_();
+      // Toggle version of hide uses fade animation effects
       this.topBar_.classList.add('fade-out');
       this.descriptionBox_.classList.add('fade-out');
-      this.controlsMode_ = LightboxControlsModes.CONTROLS_HIDDEN;
+    } }
+
+  /**
+   * Show lightbox controls.
+   * @private
+   */
+  showControls_() {
+    this.carousel_.setAttribute('controls', '');
+
+    this.topBar_.classList.remove('fade-out');
+    this.topBar_.classList.remove('hidden');
```

Comment:
this might be an existing bug, but purely using opacity to show/hide still keeps the area "tappable", worth a tests

(also normally I would say we need to also set `aria-hidden` when using opacity for hiding and showing, but honestly I think it is better to always have the controls readable by screen readers in this case, specially given there are only a few)

Reply:
(Merely  a suggestion, haven't looked at the full diff.)

Can `pointer-events: none` be an alternative in this case?

Reply:
Right, I think we need to add `visibility: hidden`.